### ISSUE TITLE
"Vehicles present" voting choice

### ISSUE THREAD

**Issue by deferred-impact** [deferred-impact/battledudes-issue-tracker#49]
"Westlands" vs "Westlands with vehicles". A rather obvious and easy-to-implement solution to "how do i let people choose if cars should be present on this map". But... why only Westlands?

Now, vehicles are a controversial issue in this community, and i fully get why. Personally i do not like them that much - getting obliterated by a car flying at you from offscreen at highway speeds is not particularly enjoyable. But i do see a need for them - cars allow large maps to be tolerable and tanks add extra complexity to the gameplay (whether that complexity is any good, is debatable)

So, why not let players choose for each map?

Now, obviously having "`$mapName`" "`$mapName` with vehicles" is not a viable solution, given how many maps we have the UI will extend beyond the screen on most monitors. But given that most players would always say "yes vehicles" or "no vehicles" no matter the map, the better solution seems to be to add another voting dimension, called "Vehicles present" and with two voting options: "yes" and "no".

So in other words, you would pick one or more maps to play, one or more gamemodes to play, and either vote yes or no for vehicles. (The "Vehicles present" option can allow multiple voting, as voting everything is the same as voting nothing in the current system)

Now, of course this would mean some extra work for Sarunas to place vehicle spawnpoints on maps that don't have them. It also does mean the possibility of Compound with vehicles - a rather silly concept. But, silly things are good, if they happen rarely
